runtime/src/accounts_index.rs

```diff
@@ -174,13 +174,157 @@ impl<T: 'static + Clone + IsCached> WriteAccountMapEntry<T> {
 }
 
 #[derive(Debug, Default)]
+pub struct RollingBitField {
+    max_width: usize,
+    min: u64,
+    max: u64, // exclusive
+    bits: Vec<u64>,
```

Comment:
We use the `BitVec` crate in other places, maybe this could just use that?

Reply:
looks like that handles the low level math and mapping, Hopefully it is fast. I have that refit to use BitVec.